docs/WritingTests.asciidoc

```diff
@@ -1408,3 +1408,13 @@ state of the specified git refspec. For example the asset caching
 functionality will override the +PRODUCTDIR+ variable. The above method can
 still be used in this case in the common case that the test schedule does not
 need to be changed or in case a custom schedule is defined by +SCHEDULE+.
+
+==Controling list of log files to upload
+
+During every job run worker produces various log files.
+By specifying +AUTOINST_UPLOAD_FROM_POOL+ variable with comma separated list of filenames
+it is possible to control which files should be uploaded to job results. Default set of files
+is defined in https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Worker/Job.pm ( +UPLOAD_FROM_POOL+ constant ).
+Keep in mind that definition of +AUTOINST_UPLOAD_FROM_POOL+ variable will overwrite default list. So in case you
```

Comment:
I suggest to turn the whole section around:

By default openQA uploads a set of file as defined in … . You can override this by specifying +AUTOINST_UPLOAD_FROM_POOL+ …

Reply:
For me it looks like I already have such structure just not exactly same wording : 
yours "By default openQA uploads a set of file as defined in "  is equal "Default set of files
is defined in " . "You can override this by specifying +AUTOINST_UPLOAD_FROM_POOL+" is equal to "definition of +AUTOINST_UPLOAD_FROM_POOL+ variable will overwrite default list ". And they going in same order as you asked . 

So I can not get what you actually mean when saying : 
> turn the whole section around 

?

Reply:
Your current text reads like this: "You can specify files. But by default you do not need to. However you can" :)

So I suggest to "turn it around" by starting with the simple case when one does not do anything, the default case. Then afterwards explain that this can be overriden. By this you also do not need to repeat again what is the order of precedence.

```suggestion
- By specifying +AUTOINST_UPLOAD_FROM_POOL+ variable with comma separated list of filenames
- it is possible to control which files should be uploaded to job results. Default set of files
- is defined in https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Worker/Job.pm ( +UPLOAD_FROM_POOL+ constant ).
- Keep in mind that definition of +AUTOINST_UPLOAD_FROM_POOL+ variable will overwrite default list.
+ By default openQA uploads a set of file as defined in the constant +UPLOAD_FROM_POOL+
+ in https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Worker/Job.pm .
+ You can override this set in the test variable +AUTOINST_UPLOAD_FROM_POOL+.
```